docs/_layout/sidebar.md

```diff
@@ -16,6 +16,7 @@
   - [Common Terms](/tips/glossary.md)
   - [Open Source Resources](/tips/OpenSourceResources.md)
   - [Open Source Tips](/tips/OpenSourceTips.md)
+  - [Glossary](/tips/glossary.md)
```

Comment:
I would add the glossary in alphabetical order 🤔 but this would be just my opinion

Reply:
> I would add the glossary in alphabetical order 🤔 but this would be just my opinion

yeah it s good in alphabetical order thanks for suggesting